delivery/services/delivery_service.py

```diff
@@ -40,7 +40,7 @@ def _run_mover(delivery_order_id, delivery_order_repo, external_program_service,
         delivery_order = delivery_order_repo.get_delivery_order_by_id(delivery_order_id, session)
         try:
 
-            cmd = ['to_outbox',
+            cmd = [path_to_mover+'/to_outbox',
```

Comment:
this will work fine.. just a note that the preferred way of constructing paths is with `os.path.join()`, e.g. `os.path.join(path_to_mover, 'to_outbox')`